### ISSUE TITLE
[sfm] 16F22: add variant with vertical stroke more to the right like 16F20

### ISSUE THREAD

**Issue by cheuk879** [silnrsi/font-shimenkan#71]
For [sfm], the vertical stroke of 16F22 should be more to the right like 16F20.

**Comment by mhosken** [comment]
samples please

**Comment by jvgaultney** [comment]
This makes sense. Only two langs use the warted cons: [hmd] and [sfm]. [hmd] uses both u16F21 and u16F22 (and not u16F20), which makes the warted/non-warted similar:
![image](https://user-images.githubusercontent.com/4226750/62858046-2a0ac580-bcf1-11e9-98cf-462f9f4c9b93.png)

However for [sfm] they use u16F20 (not u16F21) so there is currently a mismatch:
![image](https://user-images.githubusercontent.com/4226750/62858167-7f46d700-bcf1-11e9-9810-61cc0a9763b4.png)

We will need a new glyph (u16F22.sfm) and a new OT lookup to make the substitution. This will need to wait until after v1.0.

**Comment by cheuk879** [comment]
Would have been nice to make it for v1.0, but we'll relate to the users.

**Comment by jvgaultney** [comment]
Well this has been this way in the font for many months and was reported a month *after* the agreed-upon deadline for all 1.0 changes.

**Comment by cheuk879** [comment]
Yes, I know, so I've already notified them accordingly.  As I mentioned some time ago, we're really in a chicken-or-egg situation: users need a font to see what's right & what's wrong as a means to tell us what they want, but we need to know what they want before we can give them a font.  This is esp. tricky when we only have a handwritten alphabet chart to start with.  Most of the time we have to guess what they may want & ask hypothetical questions.  And it happened a week ago they responded to this guess, saying, "yes, that's what we want!"  Anyhow, we'll aim for v1.1 for this.

**Comment by jvgaultney** [comment]
Thanks - This is why I've been pressing to get 1.0 out, even with known bugs.
